activerecord/lib/active_record/enum.rb

```diff
@@ -119,8 +132,9 @@ def serialize(value)
       attr_reader :name, :mapping
     end
 
-    def enum(definitions)
+    def enum(definitions, options = {})
```

Comment:
Methods with double hashes are ugly as hell, when you have to surround the first hash with braces. Let's turn prefix into an optional keyword argument and avoid that.